features/javascript/fields/body.feature

```diff
@@ -0,0 +1,130 @@
+@javascript @stable
+Feature: Body validation
+
+  Background:
+    Given you expect "body" to equal:
+    """
+    {
+      "firstName": "John",
+      "lastName": "Locke"
+    }
+    """
+
+  Scenario: Matching body
+    Given the actual "body" equals:
+    """
+    {
+      "firstName": "John",
+      "lastName": "Locke"
+    }
+    """
+    When Gavel validates the HTTP message
+    Then the actual HTTP message is valid
+    And the result field "body" equals:
+    """
+    {
+      "valid": true,
+      "kind": "json",
+      "values": {
+        "expected": "{\n  \"firstName\": \"John\",\n  \"lastName\": \"Locke\"\n}",
+        "actual": "{\n  \"firstName\": \"John\",\n  \"lastName\": \"Locke\"\n}"
+      },
+      "errors": []
+    }
+    """
+
+  Scenario: Non-matching body
+    Given the actual "body" equals:
+    """
+    {
+      "firstName": "John"
+    }
+    """
+    When Gavel validates the HTTP message
+    Then the actual HTTP message is NOT valid
+    And the result field "body" equals:
+    """
+    {
+      "valid": false,
+      "kind": "json",
+      "values": {
+        "expected": "{\n  \"firstName\": \"John\",\n  \"lastName\": \"Locke\"\n}",
+        "actual": "{\n  \"firstName\": \"John\"\n}"
+      },
+      "errors": [
+        {
+          "message": "At '/lastName' Missing required property: lastName",
+          "location": {
+            "pointer": "/lastName",
+            "property": ["lastName"]
+          }
+        }
+      ]
+    }
+    """
+
+  # Comparing JSON to plain text produces a validation error
+  # and returns "null" kind, indicating no validation has happened.
```

Comment:
I think this can be description inside the scenario - https://relishapp.com/cucumber/cucumber/docs/gherkin/using-descriptions-to-give-features-context#everything-with-a-description

```gherkin
Scenario: Other body type
  
  Comparing JSON to plain text produces a validation error...

  Given ...
```